.github/kokoro/steps/hostsetup.sh

```diff
@@ -64,6 +64,7 @@ sudo apt-get install -y \
         python3-yaml \
         virtualenv \
         ninja-build \
+        pigz \
```

Comment:
pigz?

Reply:
I found this to be compressing library that can run on multiple cores, unlike gzip. The files to compress can become pretty huge with bigger parts, hence the choice to use this instead of gzip

Reply:
SGTM!

Maybe you want to use lzma, 7z or bzip2 instead? (Or maybe we should publish them all in the same way that the Linux kernel does?)